spec/features/user_creates_item_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+feature 'user creates item', %{
+  As a signed up user
+  I want to create new item
+  So that other users can rate and review it
+} do
+  scenario 'create succcessful item' do
+    user = FactoryGirl.create(:user)
+    item = FactoryGirl.create(:item)
```

Comment:
The `create` method actually creates an item in your database. You want the user to test this functionality. Delete this line.